src/ralph/export_to_ng/publishers.py

```diff
@@ -194,3 +195,34 @@ def sync_virtual_server_to_ralph3(sender, instance=None, created=False, **kwargs
         'parent_id': asset.id if asset else None,
         'custom_fields': _get_custom_fields(instance),
     }
+
+
+@ralph3_sync(Network, topic='sync_ipaddress_to_ralph3')
+def sync_gateway_ipaddress_to_ralph3(sender, instance=None, created=False, **kwargs):
+    if not instance.gateway:
+        return {}
+    return {
+        'id': None,
+        'address': instance.gateway,
+        'device_id': None,
+        'hostname': None,
+        'is_management': False,
+        'is_gateway': True,
+        'dhcp_expose': False
+    }
+
+
+@ralph3_sync(IPAddress)
+def sync_ipaddress_to_ralph3(sender, instance=None, created=False, **kwargs):
+    return {
```

Comment:
i don't know for sure, but maybe it'll be better to sync it as a part of `sync_device_to_ralph3`/`sync_virtual_server_to_ralph3`? 
Arguments:
* we don't want to store empty IP addresses (without assigned device) in Ralph3
* right now, scan saves IP address after every scan, so (almost) all of our ip addresses will be synced every time our networks are scanned
* we can't just check if ip has device assigned - there could be the case, where device is detached in current request (save) from ip (but we could workaround it, probably)